org.openjdk.jmc.console.ext.agent/src/main/java/org/openjdk/jmc/console/ext/agent/manager/wizards/EventEditingWizardConfigPage.java

```diff
@@ -66,13 +113,262 @@ public void createControl(Composite parent) {
+		methodNameText.setMessage(EVENT_METHOD_NAME_DESCRIPTION);
+		methodNameText.setLayoutData(gdText);
+
+		methodDescriptorText = createText(parent);
+		methodDescriptorText.setMessage(EVENT_DESCRIPTOR_DESCRIPTION);
+		methodDescriptorText.setLayoutData(gdText);
+
+		Control separator = createSeparator(parent);
+		GridData gdSeperator = new GridData(SWT.FILL, SWT.FILL, true, false);
+		gdSeperator.horizontalSpan = NUM_COLUMNS;
+		gdSeperator.heightHint = 20;
+		separator.setLayoutData(gdSeperator);
+	}
+
+	private void createOtherInfoSection(Composite parent) {
+		GridData gdLabel = new GridData(SWT.BEGINNING, SWT.CENTER, false, false);
+		GridData gdText = new GridData(SWT.FILL, SWT.CENTER, true, true);
+		gdText.minimumWidth = 0;
+		gdText.widthHint = 300;
+		gdText.horizontalSpan = NUM_COLUMNS - 1;
+
+		Label label = createLabel(parent, EVENT_PATH_LABEL);
+		label.setLayoutData(gdLabel);
+
+		pathText = createText(parent);
+		pathText.setMessage(EVENT_PATH_DESCRIPTION);
+		pathText.setLayoutData(gdText);
+
+		label = createLabel(parent, EVENT_LOCATION_LABEL);
+		label.setLayoutData(gdLabel);
+
+		locationCombo = new Combo(parent, SWT.DROP_DOWN | SWT.READ_ONLY);
+		List<String> locations = new ArrayList<>();
+		for (Location l : Location.values()) {
+			locations.add(l.name());
+		}
+		locationCombo.setItems(locations.toArray(new String[0]));
+		locationCombo.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, false));
+	}
+
+	private void createCheckBoxSection(Composite parent) {
+		Composite checkBoxTextContainer = new Composite(parent, SWT.NONE);
+		checkBoxTextContainer.setLayout(new GridLayout(2, false));
+
+		useRethrowBtn = new Button(checkBoxTextContainer, SWT.CHECK);
+		useRethrowBtn.setLayoutData(new GridData(SWT.FILL, SWT.FILL, false, false));
+
+		GridData gdLabel = new GridData(SWT.FILL, SWT.CENTER, true, true);
+
+		Label label = createLabel(checkBoxTextContainer, EVENT_RETHROW_LABEL);
+		label.setLayoutData(gdLabel);
+
+		recordStackTraceBtn = new Button(checkBoxTextContainer, SWT.CHECK);
+		recordStackTraceBtn.setLayoutData(new GridData(SWT.FILL, SWT.FILL, false, false));
+
+		label = createLabel(checkBoxTextContainer, EVENT_STACKTRACE_LABEL);
+		label.setLayoutData(gdLabel);
+
+		checkBoxTextContainer.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, false));
+	}
+
+	private Label createLabel(Composite parent, String text) {
+		Label label = new Label(parent, SWT.NONE);
+		label.setText(text);
+		return label;
+	}
+
+	private Text createText(Composite parent) {
+		Text text = new Text(parent, SWT.BORDER);
+		text.setEnabled(true);
+		return text;
+	}
+
+	private Text createTextMulti(Composite parent) {
+		Text text = new Text(parent, SWT.BORDER | SWT.MULTI | SWT.V_SCROLL);
+		text.setEnabled(true);
+		return text;
+	}
+
+	private Label createSeparator(Composite parent) {
+		return new Label(parent, SWT.SEPARATOR | SWT.HORIZONTAL);
+	}
+
+	private void bindListeners() {
+		idText.addModifyListener(new ModifyListener() {
```

Comment:
Actually, these anonymous classes can be simplified into lambda expressions.

```suggestion
		idText.addModifyListener(e -> event.setId(idText.getText()));
```